config/options.go

```diff
@@ -46,4 +46,6 @@ var (
     // Database
     DB_TYPE = MustNewStringOption("db.type", "disk", "The type of database to use.");
     DB_PG_URI = MustNewStringOption("db.pg.uri", "", "Connection string to connect to a Postgres Databse. Empty if not using Postgres.");
+
+    STALELOCK_DURATION = MustNewIntOption("stale.duration", 2, "Amount of time a lock can be unused before getting removed");
```

Comment:
See an option like `TASK_MIN_REST_SECS`:
 - We want to put the unit at the end of the variable name so coders always know the units.
 - We can do math when computing the value to make it easier for coders to see what the value actually is (e.g. `86400` may seem a little random for a time duration, but `24 * 60 * 60` is much more clear).
 - For the actual key, we want something that tells users roughly what functionality this will be tied to. "stale" is not a good top-level key". Maybe something like "lockmanager.staleduration".

Two seconds is a very short duration.
I'm thinking a couple hours.